net/tcp/tcp.h

```diff
@@ -175,7 +175,6 @@ struct tcp_conn_s
   uint8_t  rcvseq[4];     /* The sequence number that we expect to
                            * receive next */
   uint8_t  sndseq[4];     /* The sequence number that was last sent by us */
-  uint32_t rexmit_seq;    /* The sequence number to be retrasmitted */
```

Comment:
I see that rexmit logic was added by https://github.com/apache/incubator-nuttx/commit/1e07b6d528ab52dd179e6631991c683749c7635d and now it is partially removed.
@a-lunev could you please participate in the review of this change?
I see some references to RFC 6298 in the comments. @anchao could you please provide a better "state of problem" explanation of what is working not as expected?